pandas/tslib.pyx

```diff
@@ -24,6 +24,11 @@ from cpython cimport (
     PyUnicode_AsUTF8String,
 )
 
+
+cdef extern from "headers/stdint.h":
+    enum: INT64_MAX
+    enum: INT64_MIN
+
```

Comment:
ok no big deal 
ideally should be consistent across the code 
maybe just import into util.pxd and use from there?